.github/workflows/vale-linter.yml

```diff
@@ -0,0 +1,19 @@
+name: Vale Linting
+
+on: pull_request
+
+permissions:
+  contents: read
+
+jobs:
+  prose:
+    if: github.repository == 'keptn-sandbox/new-keptn-docs-engine'
```

Comment:
@iamrajiv would this condition be a problem when we move the documentation to another repo?

Reply:
Yeah @imsingh if we move to another repository we have to update this. I added this normally like I was just following some standards. I will remove it because it's not important.